services/movie.py

```diff
@@ -38,3 +40,7 @@ def create_movie(
         movie.actors.set(actors_ids)
 
     return movie
```

Comment:
You don't need return anything in create/update/delete functions, check everywhere